packages/client/test/e2e/client.test.ts

```diff
@@ -340,4 +340,26 @@ describe('Http Client', () => {
       })
     })
   })
+
+  describe('Endpoints', () => {
+    describe('Map', () => {
+      const endpoint = { channel: { name: 'telegram', version: '1.0.0' }, identity: '*', sender: 'yoyo', thread: 'ya' }
+      let convoId: string | undefined
+
```

Comment:
Add those scenarios:

- Calling listEndpoints when no mapping was created should return a 404 or empty array.
- Try creating an endpoint mapping with an invalid channel.
- Try creating an endpoint mapping with empty identity, sender, and thread.